src/Views/Menu.java

```diff
@@ -17,9 +18,9 @@ public class Menu {
     private final mySQLQuery controller = new mySQLQuery();
 
     private final String[] menuItems = {
-            "1) Get all Products in inventory",
+            "1) Get a list of all products",
```

Comment:
Spec says "List which products we currently have in inventory" so lets agree on what inventory means but this is probably correct.